ple/games/donkeykong/Donkey.py

```diff
@@ -0,0 +1,117 @@
+__author__ = 'Erilyth'
+import pygame
+from Person import Person
+
+'''
+This class defines all the Donkey Kongs present in our game.
+Each donkey kong can only move on the top floor and cannot move vertically.
+'''
+
+
+class Donkey(Person):
+    def __init__(self, raw_image, position, rng):
+        super(Donkey, self).__init__(raw_image, position, rng)
+        self.__speed = 2
+        self.rng = rng
+        self.__direction = int(self.rng.rand() * 100) % 2
+        self.__cycles = 0
+        self.__stopDuration = 0
+        self.IMAGES = {
+            "kong0": pygame.image.load('Assets/kong0.png'),
```

Comment:
performance: Might be useful to use image.convert() or image.convert_alpha() depending on if the image uses alpha. 

I should have mentioned on my first pass through. You can just iterate through the dict converting & assigning it